CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 
 * Use filter label when condition has a predicate. [#5886] by [@ko-lem]
 * Fix error when routing with array containing symbol. [#5870] by [@jwesorick]
+* Allow specifying custom input_html for DateRangeInput [#5864] by [@mirelon]
```

Comment:
Point to the PR (#5867) instead of to the closed issue, and remove the link to the issue. From the PR we already link to the closed issue.